rainbows.rb

```diff
@@ -27,3 +30,20 @@ def mix_colors(color1,color2)
 
 new_color = mix_colors(red,green)
 puts "#{color_lookup[red]} #{red} + #{color_lookup[green]} #{green} = #{color_lookup[new_color]} #{new_color}"
+
+
+rainbow = [
+  {red: [255, 0, 0]},
+  {orange: [255, 127, 0]},
+  {yellow: [255, 255, 0]},
+  {green: [0, 255, 0]},
+  {blue: [0, 0, 255]},
+  {indigo: [75, 0, 130]},
+  {electric_violet: [143, 0, 255]}
+]
```

Comment:
This works, but I actually wanted to values inside the hash to be hashes themselves. We'll go over it in class!